frontend/src/components/App/settings/hook.tsx

```diff
@@ -0,0 +1,13 @@
+import { useEffect, useState } from 'react';
+import { useTypedSelector } from '../../../redux/reducers/reducers';
+
+export const useSettings = function (settingName: string) {
+  const storeSettingNameVal = useTypedSelector(state => state.ui.settings[settingName]);
+  const [settingForSettingName, setSettingForSettingName] = useState(storeSettingNameVal);
```

Comment:
`settingForSettingName` is very confusing as a name. Maybe something like `settingsItem` or `settingsEntry`?

Reply:
fixed